integration-cli/docker_cli_build_test.go

```diff
@@ -602,6 +602,36 @@ RUN [ $(cat "/test dir/test_file6") = 'test6' ]`, command, command, command, com
 	}
 }
 
+func (s *DockerSuite) TestBuildChownFlagAdd(c *check.C) {
```

Comment:
I'm still not convinced we want to add this to `ADD`.

We definitely don't need two slow integration tests for this. The code path is 99% the same. Let's just have a single one for COPY. Even better would be to add it to an existing integration test that does COPY, but I'm not sure if we have one in the API suite yet.

`integration-cli` is deprecated, so these should be API tests. We just merged `integration/` so there isn't a lot of work there yet, but at least this should be in `docker_api_build_test.go`